ui/admin/app/components/model-providers/ModelProviderForm.tsx

```diff
@@ -238,4 +246,55 @@ export function ModelProviderForm({
             </div>
         </div>
     );
+
+    function renderCustomConfigTooltip(modelProviderId: string) {
+        const link = ModelProviderConfigurationLinks[modelProviderId];
+        return (
+            <Tooltip>
+                <TooltipTrigger asChild>
+                    <Link to={link} size="icon" variant="ghost" as="button">
+                        <CircleHelpIcon className="text-muted-foreground" />
+                    </Link>
+                </TooltipTrigger>
+
+                <TooltipContent
+                    side="right"
+                    className="bg-secondary text-foreground max-w-80"
+                >
+                    This model provider supports additional environment variable
+                    configurations. Click to learn more.
+                </TooltipContent>
+            </Tooltip>
+        );
+    }
+
+    function renderLabelWithTooltip(label: string) {
+        const tooltip =
+            ModelProviderRequiredTooltips[modelProvider.id]?.[label];
+        return (
+            <div className="flex items-center">
+                {label}
+                {tooltip && (
+                    <Tooltip>
+                        <TooltipTrigger asChild>
+                            <Button
+                                size="icon"
+                                variant="ghost"
+                                onClick={(e) => e.preventDefault()}
+                            >
+                                <CircleHelpIcon className="text-muted-foreground" />
+                            </Button>
+                        </TooltipTrigger>
+
+                        <TooltipContent
+                            side="right"
+                            className="bg-secondary text-foreground max-w-80"
+                        >
+                            {tooltip}
+                        </TooltipContent>
+                    </Tooltip>
```

Comment:
I have lots of places I'd like to repeat this pattern, can you extract this into a `composed` component?

Reply:
It'll also help to have these wrapped in custom components if (big if) we decide to add mobile view. Tooltips aren't mobile/touch friendly, so we'd need to add some behavior to open a drawer or something on tap

Reply:
Yeah of course! Will move these bits to composed